cluster/terraform_kubernetes/config/filebeat/filebeat.yml.tmpl

```diff
@@ -14,6 +14,25 @@ filebeat.autodiscover:
             enabled: false
           deployment: true
 
+# Global processors run AFTER enrichment and after all inputs
+processors:
+  - drop_fields:
+      fields:
+      - agent.ephemeral_id
+      - agent.id
+      - agent.name
+      - agent.version
+      - app.duration
+      - app.duration_ms
+      - app.pid
+      - app.tags
+      - app.thread
```

Comment:
The app fields are created in the logstash pipeline, and we should keep these anyway, so lets remove from here.